monasca/templates/api-deployment.yaml

```diff
@@ -51,11 +51,17 @@ spec:
             - name: MYSQL_HOST
               value: "{{ .Release.Name }}-mysql"
             - name: MYSQL_USER
-              value: {{ .Values.api.mysql.user | quote }}
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Release.Name }}-mysql-api-secret"
+                  key: username
             - name: MYSQL_PASSWORD
-              value: {{ .Values.api.mysql.password | quote }}
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Release.Name }}-mysql-api-secret"
+                  key: password
             - name: MYSQL_DB
-              value: {{ .Values.mysql.mysqlDatabase | quote }}
+              value: "mon"
```

Comment:
hmm actually lets leave it as the mysql.mysqlDatabase that is configurable from the users point of view we can not hide that